template/bootstrap3/bin-shop-checkout-address-selection.html

```diff
@@ -1,5 +1,6 @@
 <div ng-init="checkout.$onInit()">
-    <bin-checkout-header sub-title-key="shop.checkout.address.selection"></bin-checkout-header>
+    <bin-checkout-header></bin-checkout-header>
+    <!-- <bin-checkout-roadmap></bin-checkout-roadmap> -->
```

Comment:
vergeten comment te deleten?

Reply:
Is het wel de bedoeling dat de roadmap weg gaat?

Reply:
Ik zie trouwens dat de sub title key verdwijnt.... dat lijkt mij raar. Leg even uit.

Reply:
Checkout moet zoiets worden, vandaar dat de sub title key verdwijnt. Roadmap zijn die dots die er zijn, maar ik geraak er niet meteen aan uit omdat in de eerste stap (waar de checkout start) de stappen nog niet meteen geweten zijn. Vandaar dat die uitgecomment werd. 

Omdat de kern nu even is om die paypal icon en paypal als default selecteren. (op vraag van Suleyman)

![screen shot 2018-08-08 at 13 07 34](https://user-images.githubusercontent.com/7522178/43833738-10a7df7c-9b0c-11e8-8646-ac8f7addfb3b.png)

Reply:
Ah dus die roadmap moet terug komen?